app/js/components/groupSelect/groupSelect.jade

```diff
@@ -27,7 +27,7 @@
         a(href="#", ng-click='triggerChange(g)')
           trimmed-text(length="35", text="{{g.display_name}}")
           | &nbsp;({{g.systems.length}})
-          i.fa.fa-trash-o.pull-right.group-control.inline-left(ng-click='deleteGroup(g); $event.stopPropagation();',
```

Comment:
This was here intentionally. Without `stopPropagation` a groups is selected when the trash can icon is clicked, which is not expected. Instead the href should be removed from the hyperlink above, I think